src/tiled/editpolygontool.h

```diff
@@ -83,6 +86,13 @@ private slots:
     void startMoving();
     void updateMovingItems(const QPointF &pos,
                            Qt::KeyboardModifiers modifiers);
+
+    void addNode(const QPointF &pos, Qt::KeyboardModifiers modifiers);
+
+    void finishAddingNode();
+
```

Comment:
Coding style: I'd suggest to leave out these empty lines so that the related functions are grouped together.